http-client/src/main/java/io/airlift/http/client/HttpClientModule.java

```diff
@@ -83,12 +99,17 @@ public void configure(Binder binder)
         configBinder(binder).bindConfig(KerberosConfig.class);
         configBinder(binder).bindConfig(HttpClientConfig.class, annotation, name);
 
-        // Shared thread pool
+        // Jetty shared thread pool
         configBinder(binder).bindConfig(JettyIoPoolConfig.class);
         binder.bind(JettyIoPoolManager.class).to(SharedJettyIoPoolManager.class).in(Scopes.SINGLETON);
+        binder.bind(QueuedThreadPoolMBean.class)
+                .toProvider(QueuedThreadPoolMBeanProvider.class)
+                .in(SINGLETON);
+        newExporter(binder).export(QueuedThreadPoolMBean.class)
+                .as(generatedNameOf(QueuedThreadPool.class, "shared_" + annotation.getSimpleName()));
```

Comment:
This shared pool is global, so the name here should be just `shared` and not use the annotation. This exports the same MBean under multiple names (one per client), which I don't think is correct.